src/renderer/renderer.js

```diff
@@ -210,8 +210,8 @@ Renderer.prototype.onLatestSyncedBlock = function (event, data) {
     if (data.lastSyncBlockNumber !== null && data.lastSyncBlockNumber !== 0) {
         progress = (100 * (data.lastSyncBlockNumber - data.uploadBlockNumber) / (data.highestBlockNumber - data.uploadBlockNumber)).toFixed(0);
         this.isSynced = data.lastSyncBlockNumber >= data.highestBlockNumber - 3;
```

Comment:
@bthaile You might not need this line any longer.

Reply:
agreed, removing that line